awscli/examples/logs/associate-kms-key.rst

```diff
@@ -0,0 +1,9 @@
+**To Associate a KMS key with the log group**
+
+The following ``associate-kms-key`` example associates KMS key with log group named ``demo-log-group``. If the command succeeds, no output is returned. ::
+
+    aws logs associate-kms-key \
+        --log-group-name demo-log-group \
+        --kms-key-id ``KMS_KEY_ID_ARN``
```

Comment:
Security software installed on my system blocks any push that contain sensitive info like KMS Key ID (even an example one). Hence, replaced it with _**KMS_KEY_ID_ARN**_.